arches_lingo/src/arches_lingo/components/header/SearchDialogue.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import { ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import Button from "primevue/button";
+import Dialog from "primevue/dialog";
+
+import BasicSearch from "@/arches_lingo/components/basic-search/BasicSearch.vue";
+
+const { $gettext } = useGettext();
+
+const visible = ref(false);
+
+const SEARCH_RESULTS_PER_PAGE = 25;
+const SEARCH_RESULT_ITEM_SIZE = 38;
+</script>
+
+<template>
+    <Button
+        icon="pi pi-search"
+        :label="$gettext('Search')"
+        @click="visible = true"
+    />
+
+    <Dialog
+        v-model:visible="visible"
```

Comment:
WAVE gives me an invalid aria-labelled by warning. Looks like it's because we're not [setting the header prop or otherwise providing a label as a pass-through.](https://primevue.org/dialog/#accessibility)